### PR TITLE
feat: new email template for scheduled-execution-failed email 

### PR DESCRIPTION
**Pull request by andreas-unleash**
What it says on the box.

Relates to # [1-1687](https://linear.app/unleash/issue/1-1687/send-an-email-when-the-scheduling-fails)

Screenshot 2023-12-01 at 15 33 08

### CODE REVIEW

File: src/lib/services/email-service.ts [hunk 1]
```diff
@@ -31,6 +31,8 @@ export interface IEmailEnvelope {
 
 const RESET_MAIL_SUBJECT = 'Unleash - Reset your password';
 const GETTING_STARTED_SUBJECT = 'Welcome to Unleash';
+const SCHEDULED_EXECUTION_FAILED_SUBJECT =
+    'Unleash - Scheduled execution failed';
```
Comment (thomasheartman) [inline src/lib/services/email-service.ts:35]: I'd make the subject line a little clearer. Scheduled execution of what? How about this?

```suggestion
    'Unleash - Scheduled change request could not be applied';
```

File: src/mailtemplates/scheduled-execution-failed/scheduled-execution-failed.html.mustache [hunk 2]
```diff
@@ -0,0 +1,522 @@
...
+                    <tr>
+                        <td align="center" valign="top">
+                            <!-- BEGIN BODY // -->
+                            <table border="0" cellpadding="0" cellspacing="0" width="100%" id="templateBody">
+                                <tr>
+                                    <td valign="top" class="bodyContent" mc:edit="body_content">
+                                        <h1>Scheduled changes execution failed</h1>
+
+                                        <br />
+                                        Scheduled change request <span><a class="changeRequestLink" href="{{{ changeRequestLink }}}" target="_blank" rel="noopener noreferrer">{{{ changeRequestLink }}}</a></span> failed to apply at {{{ scheduledAt }}} due to {{{ errorMessage }}}.
```
Comment (thomasheartman) [inline src/mailtemplates/scheduled-execution-failed/scheduled-execution-failed.html.mustache:480]: Does this use the link to the change request as the title? As in https://hosted.unleash.io/bob/projects/x/change-requests/9? That feels confusing. Can we use the change request id and title (if present)?

**Review by thomasheartman — COMMENTED** [review]
Got a couple things to sort out before we send this out. I'm assuming you've talked to someone with experience regarding the templates? It seems like a lot of html, but if that's how we do it, then that's how we do it 💁🏼

File: src/mailtemplates/scheduled-execution-failed/scheduled-execution-failed.html.mustache (hunk 2 above)
Comment (andreas-unleash) [reply src/mailtemplates/scheduled-execution-failed/scheduled-execution-failed.html.mustache]: Changed: if title is present would be `id: title` if not `id`

File: src/mailtemplates/scheduled-execution-failed/scheduled-execution-failed.plain.mustache [hunk 3]
```diff
@@ -0,0 +1,6 @@
+Scheduled changes execution failed
+
+Scheduled change request {{{ changeRequestLink }}} failed to apply at {{{ scheduledAt }}} due to {{{ errorMessage }}}.
```
[the remark this answers is not in the document]
Comment (andreas-unleash) [reply src/mailtemplates/scheduled-execution-failed/scheduled-execution-failed.plain.mustache:3]: This is the plain text one.  This is what we do for the reset-password plain text (show the link)

**Review by thomasheartman — APPROVED** [review]

**Pull request merged**